newIDE/app/src/stories/componentStories/LayoutEditor/LayersList.stories.js

```diff
@@ -0,0 +1,65 @@
+// @flow
+
+import * as React from 'react';
+import { action } from '@storybook/addon-actions';
+
+// Keep first as it creates the `global.gd` object:
```

Comment:
You're sure the order is important?

Reply:
I don't know, I don't understand this comment. I only copy-paste it with the import that was in the main story.